W-18120920-Changes for FIPS compliance (#3115)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/installing-an-enterprise-license.adoc

MuleSoft makes available a trial version of the Enterprise Edition of Mule runtime engine (Mule) for the purpose of evaluation. Though perfect for exploring Mule and Anypoint Studio, the trial license for Enterprise limits usage of Mule and isn't appropriate for production uses.

Complete this steps to acquire and install a non-trial *Enterprise license* before you use Mule runtime in a production environment.

. Contact your MuleSoft account representative or http://www.mulesoft.com/contact[the MuleSoft sales team] to acquire an *Enterprise license*. Depending on your subscription, the license is provided

>   either as:

Author: ```suggestion
. Contact your MuleSoft account representative or http://www.mulesoft.com/contact[the MuleSoft sales team] to acquire an *Enterprise license*. Depending on your subscription, the license is provided either as:
```

.. A single `license.lic` file
.. A `.zip` file that contains one or more license files:

>    *** For Government Cloud customers, the `.zip` file includes both a FIPS, and a non-FIPS license.

Author: ```suggestion
   *** For Government Cloud customers, the `.zip` file includes both a `license-fips.lic` (FIPS), and a `license.lic` (non-FIPS) license.
```

+
Sometimes the license installation fails and it might be necessary to manually delete `$MULE_HOME/conf/muleLicenseKey.lic`.

* To verify the license before installing it, run:
+
`mule -verifyLicenseDetails [path_to_license]`
+
The command outputs the same information as in `mule -verifyLicense`.

== Download Your License Key File

. Log in to the MuleSoft Support portal at https://help.mulesoft.com using your credentials. If you don't have login information, contact your Customer Success Manager.
. Click on the *Support* tab located on the navigation menu of the Mulesoft Help Center home page.
. Click on the *Subscriptions* tab located halfway through the page.
. Click on the *Subscription Name* of the subscription you would like a license key for. Please note that you must click on the Subscription Name (second column on the right) or you won't be forwarded to the correct page.
. Click on the *License ID* number located at the end of the page on the right.
. Click on the *License Key* file located in the Notes & Attachments section at the beginning of the page on the right to download your license key.
.. Some subscriptions provide a direct `license.lic` file.

> .. Others provide a `.zip` file that contains one or more license files. Extract the `.zip` and use the correct license as described above.

Author: ```suggestion
.. Others provide a `.zip` file that contains one or more license files. Extract the `.zip` file and use the correct license previously described.
```
